test/powershell/Language/Scripting/NativeExecution/NativeCommandProcessor.Tests.ps1

```diff
@@ -196,19 +196,13 @@ Categories=Application;
         elseif ($IsLinux) {
             # Validate on Linux by reassociating default app for text file
             & $TestFile
-            $startTime = Get-Date
             # It may take time for handler to start
-            while (((Get-Date) - $startTime).TotalSeconds -lt 10 -and (-not (Test-Path "$HOME/nativeCommandProcessor.Success"))) {
-                Start-Sleep -Milliseconds 100
-            }
+            Wait-FileToBePresent -File "$HOME/nativeCommandProcessor.Success" -TimeoutInSeconds 10 -IntervalInMilliseconds 100 > $null
```

Comment:
Every use we add ">$null" - so maybe do this internally in Wait-FileToBePresent ? If we need we could add `-Passthrou`
And why not use Out-Null? It is optimized for pipelines.

Reply:
Will change to ignore output in `Wait-FileToBePresent`. `-PassThru` can be added when there is a need coming up.

As for Out-Null, the optimization is to make `Out-Null` and same as `> $null` when it's the last command in a pipeline, namely write to null pipe directly without going through the cmdlet.